repo-cleanup.sh

```
@@ -0,0 +1,102 @@
+set -euo pipefail
+
+DRY_RUN="${DRY_RUN:-no}"
+
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+run_or_echo() {
+  if [ "$DRY_RUN" = "yes" ]; then
+    echo "[DRY RUN] $*"
+  else
+    eval "$@"
```

Shellcheck (0.11.0)

[warning] eval negates the benefit of arrays. Drop eval to preserve whitespace/symbols (or eval as string).

(SC2294)